### ISSUE TITLE
DOC: low Contrast issue

### ISSUE THREAD

**Issue by MR-Elbarbary** [pandas-dev/pandas#60041]
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

[https://pandas.pydata.org/docs/getting_started/index.html#intro-to-pandas](url)

### Documentation problem

here in dark mode,
the text has low contrast issue, it's hard to read the test
![image](https://github.com/user-attachments/assets/59298eb6-5750-420a-9eab-fc47a7b8d75d)

### Suggested fix for documentation

```html
<div class="d-flex flex-row tutorial-card-header-2" bis_skin_checked="1">
<button class="btn btn-dark btn-sm"></button>
What kind of data does pandas handle?
</div>
```
making the text color "What kind of data does pandas handle" white or any other bright color to increase the contrast
my changing the css color to white,
If you accept to fix this issue let me fix it (it will be my first PR)

**Comment by asishm** [comment]
Thanks for the report. There's already an open issue present - see https://github.com/pandas-dev/pandas/issues/60024

**Comment by easternsun7** [comment]
Maybe take a look at this PR: Update getting_started.css #60027, this was resolved in both dev and 2.1 versions, so it seems that this task doesn't need to be discussed for now.
